permits/models.py

```diff
@@ -907,6 +909,7 @@ class WorksObjectProperty(models.Model):
         (INPUT_TYPE_TEXT, _("Texte")),
         (INPUT_TYPE_REGEX, _("Texte (regex)")),
         (INPUT_TYPE_TITLE, _("Titre")),
+        (INPUT_TYPE_FILE_DOWNLOAD, _("Fichier (à télécharger)")),
```

Comment:
Please sort choices according to their values (should be just after `(INPUT_TYPE_FILE, _("Fichier")),`)

Reply:
Done! Merci